apps/gro/views/confirm.html

```diff
@@ -1,39 +1,30 @@
 {{<layout}}
-{{$step}}{{/step}}
-
-{{$journeyHeader}}
-{{#t}}journey.header{{/t}}
-{{/journeyHeader}}
-
-{{$validationSummary}}
-{{> partials-validation-summary}}
-{{/validationSummary}}
-
-{{$propositionHeader}}
-{{> partials-navigation}}
-{{/propositionHeader}}
-
-{{$header}}{{#t}}pages.confirm.header{{/t}}{{/header}}
-
-{{$content}}
-
-<p style="margin-bottom: 3em;">{{#t}}pages.confirm.subheader{{/t}}</p>
-
-{{<partials-form}}
-
-{{$form}}
-
-{{> partials-enquiry-details}}
-
-{{> partials-order-details}}
-
-{{> partials-contact-details}}
-
-{{#input-submit}}Confirm submission{{/input-submit}}
-{{/form}}
-
-{{/partials-form}}
-
-{{/content}}
+  {{$step}}{{> partials-step}}{{/step}}
+
+  {{$journeyHeader}}
+    {{#t}}journey.header{{/t}}
+  {{/journeyHeader}}
+
+  {{$validationSummary}}
+    {{> partials-validation-summary}}
+  {{/validationSummary}}
+
+  {{$propositionHeader}}
+    {{> partials-navigation}}
+  {{/propositionHeader}}
+
+  {{$header}}{{#t}}pages.confirm.header{{/t}}{{/header}}
+
+  {{$content}}
+    <p style="margin-bottom: 3em;">{{#t}}pages.confirm.subheader{{/t}}</p>
+    {{<partials-form}}
+      {{$form}}
+        {{#tableSections}}
+          {{> partials-summary-table}}
+        {{/tableSections}}
+        {{#input-submit}}Confirm submission{{/input-submit}}
```

Comment:
Although I know it hadn't been committed to a translation prior to this change, it should be done as part of this commit. 
Please add "Confirm submission" to the translations